sg_otio/utils.py

```diff
@@ -111,7 +111,7 @@ def compute_clip_shot_name(clip):
     shot_metadata = sg_metadata.get("shot", {}) or {}
     if shot_metadata.get("code"):
         return clip.metadata["sg"]["shot"]["code"]
-    if clip.markers:
+    if clip.markers and clip.markers[0].name:
```

Comment:
Looking at the old code, may be we should do a loop on all markers? Is it a big deal to add it and return the first match?
https://github.com/GPLgithub/tk-framework-editorial/blob/f83d7f988461c0502b0f44693275a2774a89e63f/python/edl/edl.py#L94